CMakeLists.txt

```diff
@@ -1,27 +1,28 @@
-cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
-project(psi4pasture LANGUAGES C CXX VERSION 0.0.1)
+cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
+project(pasture LANGUAGES C CXX VERSION 0.0.1)
 
-# sortout transqt2/ccsort interdependence
 if(ENABLE_transqt2 OR ENABLE_ccsort)
   if(NOT (ENABLE_ccsort AND ENABLE_transqt2))
     message(STATUS  "ccsort and transqt2 enabled since interdependent")
     set(ENABLE_ccsort "ON")
     set(ENABLE_transqt2 "ON")
   endif()
 endif()
-list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
 find_package(psi4 1.0 REQUIRED)
+include(psi4PluginCache)
 
 # define the install path
-get_filename_component(psi4_lib_loc ${core_LIBRARY} DIRECTORY)
+get_filename_component(psi4_lib_loc ${psi4_LIBRARY} DIRECTORY)
 set(install_loc "${psi4_lib_loc}/driver/pasture")
```

Comment:
Don't have to decide now, but what do you think of installing to `lib/${PYMOD_INSTALL_LIBDIR}/pasture/ccsort` instead of `lib/${PYMOD_INSTALL_LIBDIR}/psi4/driver/pasture/ccsort`? If `PYTHONPATH` can find python module psi4 (which clearly it can or psi4 couldn't work), then it can find pasture.